hermes/util.py

```diff
@@ -2,71 +2,46 @@
 Project-wide utilities.
 """
 
-import collections
 import logging
-import tornado
-from jinja2 import Environment, PackageLoader
+import requests
 
 from .settings import settings
-from .version import __version__
 
 
 log = logging.getLogger(__name__)
 
-_TRUTHY = set([
-    "true", "yes", "1", ""
-])
 
+class SlackHelper(object):
+    @classmethod
```

Comment:
If you decide to keep this as a method on the class you can just make this a `@staticmethod` instead of a `@classmethod` so you don't have an unused `cls` being passed in.